src/dotnet/APIView/APIViewWeb/Client/css/site.scss

```diff
@@ -446,6 +446,21 @@ code-inner {
     cursor: pointer;
 }
 
+.btn-group {
+    .btn.active {
+        border-radius: .25rem !important;
+    }
+
+    .btn:not(.active) {
+        transform: scaleY(0.9);
```

Comment:
nit: downscaling messes with fonts a bit, consider upscaling the active one.

Reply:
Scaling either way will. I did originally consider upscaling the active one, but the buttons are already pretty big. But at a 10% scale either direction, it's not that bad. Originally I was trying to merely make the top a little higher and the text aligned, but there's so many styles (mostly from bootstrap) here that it was more difficult for an internal tool than I wanted to spend time on. Someone better at CSS3 (I stopped doing most web dev circa CSS2 debut) is welcome to improve on this. :)